src/renderer/entities/transaction/lib/index.ts

```diff
@@ -4,3 +4,6 @@ export * from './common/utils';
 export * from './common/types';
 export * from './common/constants';
 export * from './validateBalance';
+
+export { OperationError } from './common/errors';
```

Comment:
Found out that you can do this instead - `export { OperationError, type OperationErrorType } from './common/errors';`